From e687223dd08e8b7e9c5ec194dc1d38fe010ccf8e Mon Sep 17 00:00:00 2001
From: josiahdc <josiah.chapman@gmail.com>
Date: Wed, 5 Oct 2022 13:05:11 -0700
Subject: [PATCH] added requirements

---
 requirements.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 requirements.txt

diff --git a/requirements.txt b/requirements.txt
new file mode 100644
index 0000000..f678e82
--- /dev/null
+++ b/requirements.txt
@@ -0,0 +1 @@
+python-dateutil==2.8.2
\ No newline at end of file
-- 
2.39.1.windows.1

